OsmAnd/src/net/osmand/plus/search/dialogs/QuickSearchDialogFragment.java

```diff
@@ -2160,4 +2171,24 @@ private PoiUIFilter initPoiUIFilter(TopIndexFilter topIndexFilter, ProcessTopInd
 		processTopIndexAfterLoad = processAfter;
 		return null;
 	}
+
+	public void showAmenityPoints(List<Amenity> amenities) {
+		WikipediaPlugin wikiPlugin = PluginsHelper.getPlugin(WikipediaPlugin.class);
+		if(wikiPlugin != null) {
```

Comment:
We show nearby card without plugin check, but here we check for it, is it intended?